src/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,127 +1,43 @@
 import React, { useEffect, useState } from 'react';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
-import { Statistic, Row, Col, Table } from 'antd';
-// TODO: update page styling and functionality, see wireframes/NewDesignProposition/AdminWireframes/Dashboard
 
-const columns = [
-  {
-    title: 'Ticket ID',
-    dataIndex: 'ticketID',
-    filters: [
-      {
-        text: 'Example',
-        value: 'Example',
-      },
-      {
-        text: 'Jim',
-        value: 'Jim',
-      },
-      {
-        text: 'Submenu',
-        value: 'Submenu',
-        children: [
-          {
-            text: 'Green',
-            value: 'Green',
-          },
-          {
-            text: 'Black',
-            value: 'Black',
-          },
-        ],
-      },
-    ],
-    // specify the condition of filtering result
-    // here is that finding the name started with `value`
-    onFilter: (value, record) => record.ticketID.indexOf(value) === 0,
-    sorter: (a, b) => a.ticketID - b.ticketID,
-    sortDirections: ['descend'],
-  },
-  {
-    title: 'Message',
-    dataIndex: 'message',
-    filters: [
-      {
-        text: 'London',
-        value: 'London',
-      },
-      {
-        text: 'New York',
-        value: 'New York',
-      },
-    ],
+/**
+ * Khaleel Musleh
+ * Created Dashboard Compoenent and set it to display the user role, this component is for future use in displaying dashboard based on user role, each dashboard will be set differently
+ * based on user role.
+ */
 
-    onFilter: (value, record) => record.ticketType.indexOf(value) === 0,
-  },
-  {
-    title: 'Date Submitted',
-    dataIndex: 'dateSubmitted',
-    defaultSortOrder: 'descend',
-    sorter: (a, b) => a.dateSubmitted < b.dateSubmitted,
-  },
-];
+const Dashboard = () => {
+  const [role, setRole] = useState([]);
 
-function onChange(pagination, filters, sorter, extra) {
-  // console.log('params', pagination, filters, sorter, extra);
-}
-
-// TODO: make Ant Design Statistics pull ticket totals from ticket tables
-const Dashboard = props => {
-  const [tickets, setTickets] = useState([]);
+  const getAccounts = () => {
```

Comment:
Organizing the dashboard based on user roles is going to drastically simplify the development process down the line. Great work making life easier for future developers. I don't see any issues.